extensions/azurecore/src/azureResource/services/terminalService.ts

```diff
@@ -76,7 +76,7 @@ export class AzureTerminalService implements IAzureTerminalService {
 			return;
 		}
 
-		const terminalName = localize('azure.cloudShell', "Azure Cloud Shell (Preview)") + ` ${shell} (${accountDisplayName})`;
+		const terminalName = localize('azure.cloudShell', "Azure Cloud Shell (Preview)") + ` ${shell.label} (${accountDisplayName})`;
```

Comment:
Should use the localize params for this

`localize('azure.cloudShell', "Azure Cloud Shell (Preview) {0} ({1})", shell.label, accountDisplayName);`